### PR TITLE
Fixing theme toggle

### LINKED ISSUE #4148: [Bug]: Fixing the light mode toggle
### Give a brief about the bug ✍️

When the light mode toggle is clicked, the colours of the game images are inverted as well, this causes some game previews to be non satisfactory:
Screenshot 2024-06-02 193302

### What is the expected behavior? 🤔

The light mode should not consist of the game previews changing, only the background and the accent (header footer search etc) should be affected by the toggle. 

### Provide step by step information reproduce the bug 📄

I have already fixed these bugs in my local repo and also well commented them:
The bug can be fixed by conditioning the toggle to not affect the game images. 

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Jasleen1210**
The issue with the inversion of game images during light theme has been fixed. Only the game images are unaffected by the toggle and the font footer changes remain same as original. Also edited gamesData.json such that the games are in order and closed a bracket that was accidentally left open and causing error in the site.
## PR Description 📜

Fixes #4148

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [ ] I have added entries for my game in main README.md
- [ ] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
game
zone

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Jasleen1210 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by ishita-43** [comment]
Please resolve merge conflicts

**Comment by Jasleen1210** [comment]
Done! @ishita-43

File: assets/js/gamesData.json [hunk 1]
```diff
@@ -1965,24 +1965,93 @@
...
+  "393":{
+  
+    "gameTitle": "Pop My Balloon",
+    "gameUrl": "Pop_My_Balloon",
+    "thumbnailUrl": "Pop_My_Balloon.png"
+  
+  },
+  
+  "394":{ 
+=======
```
Comment (ishita-43) [inline assets/js/gamesData.json:1985]: REMOVE THIS LINE

File: assets/js/gamesData.json [hunk 2]
```diff
@@ -2069,6 +2165,13 @@
   "gameTitle" : "Colour_Generator_Game",
   "gameUrl": "Colour_Generator_Game",
   "thumbnailUrl": "Colour_Generator_Game.png"
+
+  }, "408":{
+  "gameTitle": "Pen_Pointer_Fight",
+  "gameUrl": "PenPointerFight",
+  "thumbnailUrl": "PenPointerFight.png"
+}}
+=======
```
Comment (ishita-43) [inline assets/js/gamesData.json:2174]: REMOVE THIS

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Review by ishita-43 — APPROVED** [review]
Ready to merge

**Comment by kunjgit** [comment]
Thank you  @Jasleen1210 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**